src/app/models/user.rb

```diff
@@ -16,6 +22,18 @@ def self.from_param(username)
     find_by(username: username)
   end
 
+  def following?(followee_id)
```

Comment:
I see you needed this to use it in your view, in this case you need this as a helper like what we have now `liked?` `saved?` helpers.

the name should be `follows?` I think?